### ISSUE TITLE
Panic when running server command

### ISSUE THREAD

**Issue by alberto98fx** [gohugoio/hugo#11248]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
hugo v0.115.3+extended darwin/arm64 BuildDate=unknown
</pre>

### Does this issue reproduce with the latest release?
Yes.

### Additional infos

- It doesn't happen when building 
- pulled from `brew`

### Stacktrace
```
Start building sites … 
hugo v0.115.3+extended darwin/arm64 BuildDate=unknown


                   |  EN    
-------------------+--------
  Pages            | 18508  
  Paginator pages  |  3389  
  Non-page files   |    20  
  Static files     |    71  
  Processed images |     0  
  Aliases          |   639  
  Sitemaps         |     1  
  Cleaned          |     0  

Built in 222422 ms
Watching for changes in /Users/alberto/Documents/{{site}}/alberto98fx/{{site}}/{archetypes,assets,content,data,layouts,package.json,static,themes}
Watching for config changes in /Users/alberto/Documents/{{site}}/alberto98fx/{{site}}/config.yml
fatal error: pipe failed

goroutine 1 [running]:
runtime.throw({0x104855544?, 0x14000580000?})
        runtime/panic.go:1047 +0x40 fp=0x140591fb760 sp=0x140591fb730 pc=0x102f886a0
runtime.sigNoteSetup(0x90)
        runtime/os_darwin.go:101 +0xc4 fp=0x140591fb790 sp=0x140591fb760 pc=0x102f85674
os/signal.signal_enable(0x2)
        runtime/sigqueue.go:205 +0x40 fp=0x140591fb7b0 sp=0x140591fb790 pc=0x102fba5b0
os/signal.enableSignal(...)
        os/signal/signal_unix.go:49
os/signal.Notify.func1(0x2)
        os/signal/signal.go:145 +0x7c fp=0x140591fb7d0 sp=0x140591fb7b0 pc=0x1030544ec
os/signal.Notify(0x14096693440, {0x140591fba08, 0x2, 0x102f5f298?})
        os/signal/signal.go:165 +0x1e0 fp=0x140591fb850 sp=0x140591fb7d0 pc=0x103054370
github.com/gohugoio/hugo/commands.(*serverCommand).serve(0x140008a6500)
        github.com/gohugoio/hugo/commands/server.go:923 +0x2e4 fp=0x140591fba80 sp=0x140591fb850 pc=0x10463d9b4
github.com/gohugoio/hugo/commands.(*serverCommand).Run(0x140008a6500, {0x0?, 0x0?}, 0x0?, {0x0?, 0x0?, 0x0?})
        github.com/gohugoio/hugo/commands/server.go:504 +0x1c8 fp=0x140591fbb50 sp=0x140591fba80 pc=0x10463adb8
github.com/bep/simplecobra.(*Commandeer).compile.func1(0x140005a2f00?, {0x14000509090?, 0x1?, 0x1?})
        github.com/bep/simplecobra@v0.3.2/simplecobra.go:113 +0x54 fp=0x140591fbbb0 sp=0x140591fbb50 pc=0x1032c87f4
github.com/spf13/cobra.(*Command).execute(0x140005a2f00, {0x14000509050, 0x1, 0x1})
        github.com/spf13/cobra@v1.7.0/command.go:940 +0x5c8 fp=0x140591fbcf0 sp=0x140591fbbb0 pc=0x1032bc488
github.com/spf13/cobra.(*Command).ExecuteC(0x140005a2600)
        github.com/spf13/cobra@v1.7.0/command.go:1068 +0x35c fp=0x140591fbdb0 sp=0x140591fbcf0 pc=0x1032bcbdc
github.com/spf13/cobra.(*Command).ExecuteContextC(...)
        github.com/spf13/cobra@v1.7.0/command.go:1001
github.com/bep/simplecobra.(*Exec).Execute(0x140005751e0, {0x1053e4b40?, 0x140000d0018?}, {0x140000c2190?, 0x14000084698?, 0x102f8e460?})
        github.com/bep/simplecobra@v0.3.2/simplecobra.go:155 +0xc4 fp=0x140591fbe40 sp=0x140591fbdb0 pc=0x1032c8964
github.com/gohugoio/hugo/commands.Execute({0x140000c2190, 0x2, 0x2})
        github.com/gohugoio/hugo/commands/commandeer.go:66 +0x288 fp=0x140591fbf20 sp=0x140591fbe40 pc=0x1046207f8
main.main()
        github.com/gohugoio/hugo/main.go:25 +0x64 fp=0x140591fbf70 sp=0x140591fbf20 pc=0x104646e44
runtime.main()
        runtime/proc.go:250 +0x248 fp=0x140591fbfd0 sp=0x140591fbf70 pc=0x102f8ace8
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140591fbfd0 sp=0x140591fbfd0 pc=0x102fbe764

goroutine 2 [force gc (idle), 3 minutes]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000084fa0 sp=0x14000084f80 pc=0x102f8b114
runtime.goparkunlock(...)
        runtime/proc.go:387
runtime.forcegchelper()
        runtime/proc.go:305 +0xb8 fp=0x14000084fd0 sp=0x14000084fa0 pc=0x102f8af58
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14000084fd0 sp=0x14000084fd0 pc=0x102fbe764
created by runtime.init.6
        runtime/proc.go:293 +0x24

goroutine 3 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000085760 sp=0x14000085740 pc=0x102f8b114
runtime.goparkunlock(...)
        runtime/proc.go:387
runtime.bgsweep(0x0?)
        runtime/mgcsweep.go:319 +0x110 fp=0x140000857b0 sp=0x14000085760 pc=0x102f76330
runtime.gcenable.func1()
        runtime/mgc.go:178 +0x28 fp=0x140000857d0 sp=0x140000857b0 pc=0x102f6ade8
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000857d0 sp=0x140000857d0 pc=0x102fbe764
created by runtime.gcenable
        runtime/mgc.go:178 +0x74

goroutine 4 [GC scavenge wait]:
runtime.gopark(0x13e6e8e?, 0xb624cf?, 0x0?, 0x0?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000085f50 sp=0x14000085f30 pc=0x102f8b114
runtime.goparkunlock(...)
        runtime/proc.go:387
runtime.(*scavengerState).park(0x1064e8040)
        runtime/mgcscavenge.go:400 +0x5c fp=0x14000085f80 sp=0x14000085f50 pc=0x102f7414c
runtime.bgscavenge(0x0?)
        runtime/mgcscavenge.go:633 +0xac fp=0x14000085fb0 sp=0x14000085f80 pc=0x102f7472c
runtime.gcenable.func2()
        runtime/mgc.go:179 +0x28 fp=0x14000085fd0 sp=0x14000085fb0 pc=0x102f6ad88
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14000085fd0 sp=0x14000085fd0 pc=0x102fbe764
created by runtime.gcenable
        runtime/mgc.go:179 +0xb8

goroutine 18 [finalizer wait, 3 minutes]:
runtime.gopark(0x1a0?, 0x1064e8e00?, 0x40?, 0xa3?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000084580 sp=0x14000084560 pc=0x102f8b114
runtime.runfinq()
        runtime/mfinal.go:193 +0x10c fp=0x140000847d0 sp=0x14000084580 pc=0x102f69e7c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000847d0 sp=0x140000847d0 pc=0x102fbe764
created by runtime.createfing
        runtime/mfinal.go:163 +0x84

goroutine 19 [GC worker (idle)]:
runtime.gopark(0xe0410de10b80?, 0x3?, 0x89?, 0x5?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000080740 sp=0x14000080720 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x140000807d0 sp=0x14000080740 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000807d0 sp=0x140000807d0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 20 [GC worker (idle)]:
runtime.gopark(0x10652d6e0?, 0x1?, 0x90?, 0x27?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000080f40 sp=0x14000080f20 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x14000080fd0 sp=0x14000080f40 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14000080fd0 sp=0x14000080fd0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 21 [GC worker (idle)]:
runtime.gopark(0xe0410e0953ad?, 0x3?, 0x1f?, 0xf6?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000081740 sp=0x14000081720 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x140000817d0 sp=0x14000081740 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000817d0 sp=0x140000817d0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 22 [GC worker (idle)]:
runtime.gopark(0x1053c96c0?, 0x14000182020?, 0x1a?, 0x14?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000081f40 sp=0x14000081f20 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x14000081fd0 sp=0x14000081f40 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14000081fd0 sp=0x14000081fd0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 5 [GC worker (idle)]:
runtime.gopark(0xe040fd4ceb3f?, 0x3?, 0xa2?, 0x60?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000086740 sp=0x14000086720 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x140000867d0 sp=0x14000086740 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000867d0 sp=0x140000867d0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 23 [GC worker (idle)]:
runtime.gopark(0xe0410de776bb?, 0x3?, 0x71?, 0x2?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000082740 sp=0x14000082720 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x140000827d0 sp=0x14000082740 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000827d0 sp=0x140000827d0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 6 [GC worker (idle)]:
runtime.gopark(0xe0410df6b084?, 0x3?, 0x3f?, 0x44?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x14000086f40 sp=0x14000086f20 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x14000086fd0 sp=0x14000086f40 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14000086fd0 sp=0x14000086fd0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 34 [GC worker (idle)]:
runtime.gopark(0xe0410e09be9c?, 0x3?, 0x55?, 0x71?, 0x0?)
        runtime/proc.go:381 +0xe4 fp=0x1400018a740 sp=0x1400018a720 pc=0x102f8b114
runtime.gcBgMarkWorker()
        runtime/mgc.go:1275 +0xec fp=0x1400018a7d0 sp=0x1400018a740 pc=0x102f6cd2c
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x1400018a7d0 sp=0x1400018a7d0 pc=0x102fbe764
created by runtime.gcBgMarkStartWorkers
        runtime/mgc.go:1199 +0x28

goroutine 7 [select]:
runtime.gopark(0x1400018cf78?, 0x3?, 0xa0?, 0xb?, 0x1400018cf62?)
        runtime/proc.go:381 +0xe4 fp=0x1400018ce00 sp=0x1400018cde0 pc=0x102f8b114
runtime.selectgo(0x1400018cf78, 0x1400018cf5c, 0x140007aed00?, 0x0, 0x0?, 0x1)
        runtime/select.go:327 +0x690 fp=0x1400018cf20 sp=0x1400018ce00 pc=0x102f9c350
go.opencensus.io/stats/view.(*worker).start(0x140007aed00)
        go.opencensus.io@v0.24.0/stats/view/worker.go:292 +0x88 fp=0x1400018cfb0 sp=0x1400018cf20 pc=0x103c28b38
go.opencensus.io/stats/view.init.0.func1()
        go.opencensus.io@v0.24.0/stats/view/worker.go:34 +0x28 fp=0x1400018cfd0 sp=0x1400018cfb0 pc=0x103c27e68
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x1400018cfd0 sp=0x1400018cfd0 pc=0x102fbe764
created by go.opencensus.io/stats/view.init.0
        go.opencensus.io@v0.24.0/stats/view/worker.go:34 +0xb0

goroutine 488670 [syscall]:
syscall.syscall6(0x140af66f4a0?, 0x140af66f890?, 0x140af66fc50?, 0x140af278030?, 0x140af2783f0?, 0x140af2787b0?, 0x140af278b70?)
        runtime/sys_darwin.go:45 +0x68 fp=0x140335853d0 sp=0x14033585310 pc=0x102fbadd8
golang.org/x/sys/unix.kevent(0x140af5329c0?, 0x140af532d80?, 0x140af533140?, 0x140af533500?, 0x140af5338c0?, 0x140af533cb0?)
        golang.org/x/sys@v0.9.0/unix/zsyscall_darwin_arm64.go:276 +0x58 fp=0x14033585440 sp=0x140335853d0 pc=0x1032f7148
golang.org/x/sys/unix.Kevent(0x140af520840?, {0x0?, 0x140af520fc0?, 0x140af521380?}, {0x14033585660?, 0x140af521b30?, 0x140af521ef0?}, 0x140aef742d0?)
        golang.org/x/sys@v0.9.0/unix/syscall_bsd.go:398 +0x40 fp=0x14033585480 sp=0x14033585440 pc=0x1032f6f70
github.com/fsnotify/fsnotify.(*Watcher).read(0x140aef75590?, {0x14033585660?, 0x140aef75d10?, 0xa})
        github.com/fsnotify/fsnotify@v1.6.0/backend_kqueue.go:702 +0x48 fp=0x140335854e0 sp=0x14033585480 pc=0x104240538
github.com/fsnotify/fsnotify.(*Watcher).readEvents(0x14011bf7420)
        github.com/fsnotify/fsnotify@v1.6.0/backend_kqueue.go:487 +0x94 fp=0x140335857b0 sp=0x140335854e0 pc=0x10423f2d4
github.com/fsnotify/fsnotify.NewWatcher.func1()
        github.com/fsnotify/fsnotify@v1.6.0/backend_kqueue.go:155 +0x28 fp=0x140335857d0 sp=0x140335857b0 pc=0x10423d9e8
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140335857d0 sp=0x140335857d0 pc=0x102fbe764
created by github.com/fsnotify/fsnotify.NewWatcher
        github.com/fsnotify/fsnotify@v1.6.0/backend_kqueue.go:155 +0x1d4

goroutine 488830 [select]:
runtime.gopark(0x14033581778?, 0x3?, 0x40?, 0xae?, 0x1403358173a?)
        runtime/proc.go:381 +0xe4 fp=0x140335815b0 sp=0x14033581590 pc=0x102f8b114
runtime.selectgo(0x14033581778, 0x14033581734, 0x140b05836b0?, 0x0, 0x140b0583e30?, 0x1)
        runtime/select.go:327 +0x690 fp=0x140335816d0 sp=0x140335815b0 pc=0x102f9c350
github.com/gohugoio/hugo/watcher.(*Batcher).run(0x14058ac8ed0)
        github.com/gohugoio/hugo/watcher/batcher.go:64 +0xa4 fp=0x140335817b0 sp=0x140335816d0 pc=0x1045f8064
github.com/gohugoio/hugo/watcher.New.func1()
        github.com/gohugoio/hugo/watcher/batcher.go:55 +0x28 fp=0x140335817d0 sp=0x140335817b0 pc=0x1045f7f88
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140335817d0 sp=0x140335817d0 pc=0x102fbe764
created by github.com/gohugoio/hugo/watcher.New
        github.com/gohugoio/hugo/watcher/batcher.go:55 +0x244

goroutine 488832 [select]:
runtime.gopark(0x14033581f18?, 0x3?, 0xd0?, 0x8?, 0x14033581eba?)
        runtime/proc.go:381 +0xe4 fp=0x14033581d40 sp=0x14033581d20 pc=0x1
runtime.selectgo(0x14033581f18, 0x14033581eb4, 0x140e1e7b1a0?, 0x0, 0x140e1e7b920?, 0x1)
        runtime/select.go:327 +0x690 fp=0x14033581e60 sp=0x14033581d40 pc=0x102f9c350
github.com/gohugoio/hugo/livereload.(*hub).run(0x1064e68e0)
        github.com/gohugoio/hugo/livereload/hub.go:39 +0x94 fp=0x14033581fb0 sp=0x14033581e60 pc=0x1045f5494
github.com/gohugoio/hugo/livereload.Initialize.func1()
        github.com/gohugoio/hugo/livereload/livereload.go:108 +0x28 fp=0x14033581fd0 sp=0x14033581fb0 pc=0x1045f5a08
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x14033581fd0 sp=0x14033581fd0 pc=0x102fbe764
created by github.com/gohugoio/hugo/livereload.Initialize
        github.com/gohugoio/hugo/livereload/livereload.go:108 +0x40

goroutine 488831 [select]:
runtime.gopark(0x140000877a8?, 0x2?, 0x18?, 0x76?, 0x1400008772c?)
        runtime/proc.go:381 +0xe4 fp=0x140000875a0 sp=0x14000087580 pc=0x102f8b114
runtime.selectgo(0x140000877a8, 0x14000087728, 0x0?, 0x0, 0x0?, 0x1)
        runtime/select.go:327 +0x690 fp=0x140000876c0 sp=0x140000875a0 pc=0x102f9c350
github.com/gohugoio/hugo/commands.(*hugoBuilder).newWatcher.func2()
        github.com/gohugoio/hugo/commands/hugobuilder.go:373 +0xb4 fp=0x140000877d0 sp=0x140000876c0 pc=0x10462ad44
runtime.goexit()
        runtime/asm_arm64.s:1172 +0x4 fp=0x140000877d0 sp=0x140000877d0 pc=0x102fbe764
created by github.com/gohugoio/hugo/commands.(*hugoBuilder).newWatcher
        github.com/gohugoio/hugo/commands/hugobuilder.go:371 +0x458
```

**Comment by alberto98fx** [comment]
Still happening :(

**Comment by jmooring** [comment]
This is fixed in v0.123.0.
